buildSrc/src/main/java/Ktor.kt

```diff
@@ -0,0 +1,7 @@
+object Ktor {
```

Comment:
This is how I ultimately want our dependencies structured.  Deps file was already getting too unwieldy